DashJSInTypeScriptTut/Using-DashJS-with-TypeScript-in-VSCode.md

```diff
@@ -1,116 +1,117 @@
+---
 summary: Using DashJS with TypeScript in VSCode
 id: Using-DashJS-with-TypeScript-in-VSCode
 categories: Typescript
 tags: medium
 status: Published
 authors: Rion, Schlomon
 Feedback Link: example.com
+---
 
 # How to use DashJS with TypeScript in VSCode
 
-
-
 <!-- ------------------------ -->
 ## Overview
+
 Duration: 1
 
 ### What You'll learn
-* How to setup a project which is able to interact with the Dash network
-* How to create and manage a Wallet (e.g. checking balance, sending Dash, etc)
-
 
+* How to set up a project which is able to interact with the Dash network
+* How to create and manage a Wallet (e.g. check a balance, send Dash, etc)
 
 <!-- ------------------------ -->
-## Npm setup
+## NPM setup
+
 Duration: 2
 
-Create a new project folder
-```
+Create a new project folder:
+
+```sh
 mkdir dashTS_example
 cd dashTS_example
 ```
 
 You can open it in VSCode now:
-```
+
+```sh
 code .
 ```
 
 In VSCode, open the terminal with `Ctrl` + `
 *(Alternatively you can use the "normal" terminal from the previous steps)*
 
-Then init the npm project with
-```
+Then initialize the NPM project:
+
+```sh
 npm init
 ```
 
-And install DashJS
-- Globally
-    `npm i dash -g`
-- Locally
-    ` npm i dash -D`
-
+And install DashJS:
 
+* Globally: `npm install --global dash`, or
+* Locally: `npm install --save-dev dash`
 
 <!-- ------------------------ -->
 ## TypeScript setup
+
 Duration: 2
 
-If TypeScript is not installed already you can install it
-- Globally:
-    `npm i typescript -g`
-- Locally (for this project only
-    `npm i typescript -D`
+If TypeScript is not installed already, install it:
+
+* Globally: `npm install --global typescript`, or
+* Locally (for this project only): `npm install --save-dev typescript`
 
 Also create an output folder for later:
-```
+
+```sh
 mkdir dist
 ```
 
-Then run the following to complete the setup
-```
+Then run the following to complete the setup:
+
+```sh
 npx tsc --init
 ```
-You can also create a file called app.ts for later use.
-
 
+You can also create a file called app.ts for later use.
```

Comment:
Instead of app.ts we should put `app.ts`
Fixing this in your pr is easier since git is treating this as a newly created line.

Reply:
Fixed in latest commit